### PR TITLE
Adding Do me a favour Feb song

### PR DESCRIPTION
**Pull request by escobartc**
## Describe your changes
The February song "Do me a favour" was added and the related files (writer, index) were updated.
## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] Someone has reviewed my code

### CODE REVIEW

File: app/index.md [hunk 1]
```diff
@@ -8,16 +8,4 @@ Welcome to _Onlyrics_, the only place on the planet where your ears won't be dis
...
-[Imagine](song/jan/vanilla-panna-cotta.md)
-
-### [Queen](writer/queen.md) | queen@onlyrics.magazine
-
-[We Are The Champions](song/jan/we_are_the_champions.md)
-
-### [The Killers](writer/the_killers) | the.killers@onlyrics.magazine
-
-[Mr. Brightside](song/jan/mr_brightside.md)
+[Do me a favour](song/feb/do_me_a_favour.md)
```
Comment (juromerop) [inline app/index.md:11]: Really nice song for February.

**Comment by josanchezva** [comment]
Looks good! I think it is ready to merge.

File: app/song/feb/do_me_a_favour.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+This is a song about a breakup.
+
+Source: https://www.songfacts.com/facts/arctic-monkeys/do-me-a-favour
+
+## Album
+Album: Favourite Worst Nightmare(2007)
+
+# SongFacts
+
+This song describes a tempestuous breakup while driving a girl home. Alex Turner has admitted it came from personal experience. He tells NME: "It's about a goodbye, really, and about me being a bit of a knob. Perhaps I were craving to experience something else and looking back and feeling like you were a bit of a knobhead, just in how you perhaps treated that person. It's just describing a goodbye. That's another thing, when you're with someone they seem happier in photos before you met her, or happier in stories from before. I always think they do."
```
Comment (juromerop) [inline app/song/feb/do_me_a_favour.md:12]: Awesome, don't stop working hard.

**Review by josanchezva — APPROVED** [review]
Ready to merge!

**Review by juromerop — APPROVED** [review]
Nice job, approved.

**Pull request merged**